src/zenml/cli/base.py

```diff
@@ -433,3 +437,80 @@ def _prompt_email(event_source: AnalyticsEventSource) -> bool:
         )
 
     return False
+
+
+@cli.command(
+    "info", help="Show information about the current user setup.", hidden=True
+)
+@click.option(
+    "--packages",
+    "-p",
+    multiple=True,
+    help="Select specific installed packages.",
+    type=str,
+)
+@click.option(
+    "--all",
+    "-a",
+    is_flag=True,
+    default=False,
+    help="Output information about all installed packages.",
+    type=bool,
+)
+@click.option(
+    "--file",
```

Comment:
Should we let users pass a string here for the actual file path?

Reply:
Refactored to allow this. I was in two minds about it, since it will generally always be us who are asking users to use this command (given that it is hidden) but I think it can't hurt.